components/button/button_default.story.vue

```diff
@@ -26,18 +26,20 @@
       v-if="icon"
       slot="icon"
     >
-      <component :is="icon" />
+      <dt-icon
+        :name="icon"
+        size="200"
```

Comment:
Same as Avatar. Conditionals for button icons:

```
BUTTON SIZE
xs → icon size 200;
sm → icon size 200; // yes, xs *and* sm are 200
md → icon size 300;
lg → icon size 400;
xl → icon size 500;
```

<img width="115" alt="image" src="https://user-images.githubusercontent.com/1165933/202039268-c356dca6-3c49-4b29-b36c-c6bcf8cd2c56.png">